test-bench-utils/lib/sinks/ssrf.js

```diff
@@ -67,9 +70,10 @@ exports.fetch = async function makeFetchRequest(params) {
  */
 exports.request = async function makeRequestRequest(params) {
   const url = formatUrl(params);
-  return new Promise((resolve, reject) => {
+  new Promise((resolve, reject) => {
```

Comment:
Hmm so now this fails locally and in the cloud right?  I don't think that's really the state we want to end up at, if it was just the cloud that would be one thing.  I think we probably want to shift and come up w/ a route that works (or just make a more aggressive timeout so we're not waiting on these requests).

I have a few ideas of how we could do this - but one maybe easy one - we know the app itself is up and running we should just be able to make a request to itself.  we should be able to pass `localhost:3000` as the input and it should work in both the cloud env and locally.